examples/SuiteSparse/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2017, NeroBurner
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-SuiteSparse)
+
+hunter_add_package(SuiteSparse)
+
+string(COMPARE EQUAL "${SuiteSparse_LICENSES}" "" is_empty)
+if(is_empty)
+  message(FATAL_ERROR "Licenses not found")
+endif()
+
+message("SuiteSparse licenses:")
+foreach(x ${SuiteSparse_LICENSES})
+  message("* ${x}")
+  if(NOT EXISTS "${SuiteSparse_LICENSES}")
+    message(FATAL_ERROR "File not found")
+  endif()
+endforeach()
+
+find_package(SuiteSparse CONFIG REQUIRED)
+
+# Test double library creation
+find_package(SuiteSparse CONFIG REQUIRED)
+
+# Create executable
+include(${USE_SuiteSparse})
+add_executable(foo foo.cpp)
+target_link_libraries(foo PRIVATE ${SuiteSparse_LIBRARIES})
```

Comment:
yes. The template for the Use file is defined here https://github.com/jlblancoc/suitesparse-metis-for-windows/blob/master/UseSuiteSparseForInstall.cmake.in

it also calls `include_directories(${SuiteSparse_INCLUDE_DIRS})`

I think the suitesparse-metis-for-windows author was inspired by the Eigen3Config and UseEigen3 cmake files